drivers/builtin/include/mbedtls/config_adjust_legacy_crypto.h

```diff
@@ -114,50 +120,109 @@
 #if defined(MBEDTLS_PSA_CRYPTO_C)
 
 #if defined(MBEDTLS_PSA_ACCEL_ALG_MD5)
+#define MBEDTLS_MD_CAN_MD5
```

Comment:
`MBEDTLS_MD_CAN_xxx` no longer exists in TF-PSA-Crypto.

(Ok, it's technically harmless to define it, but it's confusing, since it won't be used.)